shell/core/plugins.js

```diff
@@ -469,3 +469,16 @@ export default function(context, inject, vueApp) {
     vueApp
   );
 }
+
+export function getProviders(context) {
```

Comment:
this should be in the plugins object along with the other $plugin related functions

Reply:
The context passed in will be of type ClusterProvisionerContext, best to mention that in the jsdoc for this function